src/ol/mapbrowserevent.js

```diff
@@ -253,21 +264,24 @@ ol.MapBrowserEventHandler.prototype.emulateClickLegacyIE_ =
 ol.MapBrowserEventHandler.prototype.emulateClick_ = function(pointerEvent) {
   var newEvent;
   newEvent = new ol.MapBrowserPointerEvent(
-      ol.MapBrowserEvent.EventType.CLICK, this.map_, pointerEvent);
+      ol.MapBrowserEvent.EventType.CLICK, this.map_, pointerEvent,
+      this.dragging_);
```

Comment:
`dragging` is only set for `POINTERDRAG` and `POINTERMOVE`? Sounds reasonable. I will change it.